torch/csrc/jit/runtime/operator.cpp

```diff
@@ -268,7 +269,7 @@ bool printerHasSpecialCaseFor(Symbol sym) {
   };
 
   return handled.count(sym) || unneeded.count(sym) ||
-      !required_namespaces.count(sym.ns());
+      !required_namespaces.count(sym.ns()) || IsFusionSubgraphKind(sym);
```

Comment:
nit - could you add a comment on why this is not needed? (i.e. `optimization pass adds it`)

Reply:
also this is causing build failures on the mobile builds